benchmark.js

```diff
@@ -46,7 +46,7 @@ const blogFour = bunyan.createLogger({
 const max = 10
 const run = bench([
   function benchBunyanTen (cb) {
-    for (var i = 0; i < max; i++) {
+    for (let i = 0; i < max; i++) {
```

Comment:
Are we good with `let` here?

Reply:
Kind of, it's applied to all benchmarks so it's not a much bigger difference.